src/Npgsql/PostgresEnvironment.cs

```diff
@@ -45,15 +48,15 @@ static class PostgresEnvironment
 
         internal static string? TargetSessionAttributes => Environment.GetEnvironmentVariable("PGTARGETSESSIONATTRS");
 
-        static string GetHomeDir()
-        {
-            var envVar = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "APPDATA" : "HOME";
-            return Environment.GetEnvironmentVariable(envVar) is string homedir
+        static string? GetHomeDir()
+            => Environment.GetEnvironmentVariable(RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "APPDATA" : "HOME") is string
```

Comment:
Is there actually any point in checking whether `GetEnvironmentVariable` returned null here?

Reply:
You're right, simplified.